botocore/docs/bcdoc/docstringparser.py

```diff
@@ -11,10 +11,12 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 from html.parser import HTMLParser
+from itertools import zip_longest
 
 PRIORITY_PARENT_TAGS = ('code', 'a')
 OMIT_NESTED_TAGS = ('span', 'i', 'code', 'a')
 OMIT_SELF_TAGS = ('i', 'b')
+HTML_BLOCK_DISPLAY_TAGS = ['p', 'note', 'ul', 'li']
```

Comment:
nit; is there a reason for making this a list and not a tuple like the others? Do we expect this to be mutated?